Ginger/GingerCoreNET/LiteDBFolder/LiteDbReportBase.cs

```diff
@@ -205,9 +225,16 @@ public class LiteDbActivity : LiteDbReportBase
     {
         public string ActivityGroupName { get; set; }
         public List<LiteDbAction> ActionsColl { get; set; }
+        public int ChildExecutableItemsCount { get; set; }
+        public int ChildExecutedItemsCount { get; set; }
+        public int ChildPassedItemsCount { get; set; }
+
         public LiteDbActivity()
         {
             ActionsColl = new List<LiteDbAction>();
+            //ChildExecutableItemsCount = new Dictionary<string, int>();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/app/Ginger-Automation/Ginger/pullRequest?prid=4549121)